app/Services/ProjectService.php

```
@@ -379,15 +379,20 @@ public function calculateDirectCostDistribution($tasks)
                 $result[$label] = [
-                    'total (PHP)' => number_format($total, 2),
-                    'percent'     => number_format($percent, 2) . '%',
+                    'total_php' => number_format($total, 2),
+                    'total_percent'     => number_format($percent, 2) . '%',
+                ];
+            } else {
+                $result[$label] = [
+                    'total_php' => 0.00,
+                    'total_percent' => '0.00%',
                 ];
             }
```

PHPMD (2.15.0)

The method calculateDirectCostDistribution uses an else expression. Else clauses are basically not necessary and you can simplify the code by not using them. (undefined)

(ElseExpression)